### PR TITLE
NCIPPE-315 | Mocha Admin does not receive an email notification when a

### PR DESCRIPTION
**Pull request by akavalloor**
Biomarker test is uploaded

Following changes were incorporated
1. CRC and Providers for the participant receives an email
2. Participant receives an email
3. System Notification for CRC and Providers
4. System Notification for Participant

### CODE REVIEW

File: src/main/java/gov/nci/ppe/services/impl/AmazonS3ServiceImpl.java [hunk 1]
```diff
@@ -461,36 +462,53 @@ public Code getCodeforFileType(String fileTypeCodeName) {
...
-				replaceThisStringForSubject, replaceSubjectStringWith);
-
-		return emailLogService.sendEmailNotification(emailId, emailServiceConfig.getSenderEmailAddress(), subject,
-				updatedTextBody, updatedTextBody);
-	}
+	private void sendEmailToCRCAndProvidersAfterFileUpload(Map<String, String> emailIds, String patientFullName, String patientId, String actionFor) {
+		if (StringUtils.isNotBlank(actionFor)
+				&& FileType.PPE_FILETYPE_ECONSENT_FORM.getFileType().equalsIgnoreCase(actionFor)) {
+			emailIds.forEach((name, emailId) -> {
+				emailLogService.sendEmailToCRCAndProvidersAfterUploadingBioMarkerReport(name, emailId, patientFullName,patientId);
```
Comment (debosarkar) [inline src/main/java/gov/nci/ppe/services/impl/AmazonS3ServiceImpl.java]: This should be the method for eConsent NOT biomarker

File: src/main/resources/EmailService.properties [hunk 2]
```diff
@@ -1,10 +1,17 @@
 email.use.aws=${USE_AWS_SES}
 sender.email.address=${EMAIL_ADDRESS}
 
+<<<<<<< Updated upstream
```
Comment (debosarkar) [inline src/main/resources/EmailService.properties]: This file is not merged. Conflicts still present

Comment (akavalloor) [reply src/main/resources/EmailService.properties]: Made the changes

File: src/main/java/gov/nci/ppe/services/impl/AmazonS3ServiceImpl.java (hunk 1 above)
Comment (akavalloor) [reply src/main/java/gov/nci/ppe/services/impl/AmazonS3ServiceImpl.java]: Will make these changes as part of another commit / story

**Pull request merged**